src/vector/init.tsx

```diff
@@ -115,6 +115,16 @@ export async function loadLanguage() {
     }
 }
 
+export async function loadSpellCheckLanguages() {
+    const langs = SettingsStore.getValue("spell-check-languages", null, false);
+
+    try {
+        await languageHandler.setSpellCheckLanguages(langs);
```

Comment:
If Electron's supposedly remembering this, do we need to set it at startup at all?